build.sbt

```diff
@@ -31,9 +31,9 @@ unmanagedJars in Compile += {
 }
 
 resolvers ++= Seq(
-  "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots",
-  "Sonatype OSS Releases" at "https://oss.sonatype.org/content/repositories/releases",
-  "uuverifiers" at "http://logicrunch.research.it.uu.se/maven"
+  ("Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots").withAllowInsecureProtocol(true),
+  ("Sonatype OSS Releases" at "https://oss.sonatype.org/content/repositories/releases").withAllowInsecureProtocol(true),
```

Comment:
Is this change necessary? These repos seem to be using 'https' which isn't an insecure protocol.

Reply:
No you're right, it's fixed now thanks.